### DISCUSSION TITLE
Where can I obtain the integrated variant setup file?

### DISCUSSION THREAD

**Discussion by ditchorama** [MiKTeX/miktex#822] [Q&A]
I need the integrated variant to allow silent uninstall of miktex however the standalone version doesn't support uninstall and I'm unable to located a source for the integrated variant (miktexsetup)

**Comment by edocevoli** [comment]
The integrated variant is a part of the MiKTeX installation. That said, you have to install MiKTeX in order to be able to uninstall it.

**Reply by ditchorama** [reply]
Previously the standalone/cmd line could do the install & Uninstall of the packages etc, the uninstall seems to have been removed now

_OLDER VERSION_
MiKTeX Setup Utility (Standalone) 2.9.7409 (MiKTeX 2.9.7440 64-bit)
Copyright (C) 2014-2020 Christian Schenk
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
miktexsetup.exe download|install|finish|factoryreset|uninstall
------------------------------------------------------
_NEWER VERSION_
MiKTeX Setup Utility (Standalone) 4.1 (MiKTeX 21.1)
© 2014-2021 Christian Schenk
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
miktexsetup_standalone.exe download|install
